plugins/http2_adapter/pubspec.yaml

```diff
@@ -1,6 +1,12 @@
 name: dio_http2_adapter
-description: An adapter that combines HTTP/2 and dio. Supports reuse connections, header compression, etc.
 version: 2.3.1
+
+description: An adapter that combines HTTP/2 and dio. Supports reuse connections, header compression, etc.
+topics:
+  - dio
+  - http2
+  - native
```

Comment:
I think this is wrong here

Reply:
In which perspective is wrong? The adapter is only for io, so I wrote "native".

Reply:
Ok, I think of native being Android/iOS code. Fine by me just wanted to make sure this was not accidental.

Reply:
Yeah sure, thanks for the attention. 👍